epuck_ros2/epuck_ros2/camera_calibration.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 1996-2020 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A temporary, very simple, solution to calibrate camera.
+# It should be delete as soon as `camera_calibration` is ready for ROS2.
+
+
+import rclpy
+from rclpy.node import Node
+from sensor_msgs.msg import Image
```

Comment:
This script is very good, but I am just wondering if using ROS here is not a bit overkilled.
Wouldn't it be simpler just to create a simple python script without any ROS dependency that would read the image directly from the camera by itself (since their is already a dependency on opencv, opencv has a module for doing this, this is maybe not as efficient as what you implemented in the camera node, but since it is only used for calibration it doesn't really matter)?

Reply:
Hmm, I didn't consider running the calibration onboard. This way I have a nice preview of the images I am taking. Also I afraid of running `findChessboardCorners()` and especially `calibrateCamera()` since those two can be really demanding even for my PC (takes a few seconds depending on a number of images).

Reply:
You're right, been able to run the calibration offboard is a good reason :smiley: